technical-vision.md

```diff
@@ -1,9 +1,28 @@
 # OpenSSF Technical Vision
 
-We envision a future where participants in the open source ecosystem use and share high quality software, with security handled proactively, by default, and as a matter of course:
-- Developers can easily learn secure development practices and are proactively guided by their tools to apply those practices and automatically informed when action is needed to prevent, remediate, or mitigate security issues.
-- Developers, auditors, and regulators can create and easily distribute security policies that are enforced through tooling and automation, providing continuous assurance of the results.
-- Developers and researchers can identify security issues (including unintentional vulnerabilities and malicious software) and have this information swiftly flow backwards through the supply chain to someone who can rapidly address the issue.
-- Community members can provide information and notifications about product defects, mitigations, quality, and supportability and have this information rapidly flow forward across the ecosystem system to all users, and users can rapidly update their software or implement mitigations as appropriate.
+An aspirational view of the future we seek to create in 2-5 years:
 
-The Open Source Security Foundation (OpenSSF) provides tools, services, training, infrastructure, and resources to achieve this vision.
+### OpenSSF is a well-respected strategic partner to large OSS foundations and individual OSS projects
+- OpenSSF is clearly positioned within broader ecosystem with a clear, altruistic charter
+- Periodically review the value proposition of OpenSSF with key stakeholders
+  - Joint OKRs on shared execution, utilization of OpenSSF-managed tools & content
+  - Roadmap alignment: if overlap exists between OpenSSF and another stakeholder, either publicly document rationale OR create/document/execute a convergence plan
+- OpenSSF speaks with a trusted voice to users, contributors, and the broader market
+
+### OpenSSF is an influential advocate for mutually-beneficial external efforts and an educator of policy decision makers
+- Support creation & use of meaningful, actionable standards (e.g. SBOM formats, MFA)
+- OpenSSF advocates for various actors (including maintainers, contributors, and adopters) in the open source ecosystem to improve their default security posture, and catalyzes efforts to reduce or eliminate friction in achieving that state; for example (but not exclusively):
+  - SLSA ensures tampering and other security threats to the software supply chain of open source projects, groups, and communities are reduced and/or eliminated
+  - Memory Safety efforts that aim to eliminate entire classes of security threats through using memory safe programming languages
```

Comment:
I would suggest softening this a bit:
```suggestion
  - Memory Safety efforts that aim to eliminate entire classes of security threats through using memory safe programming languages where possible
```

Reply:
i like this addition, since we can't convert everything.  "where possible" allows us room for alternatives